inference-engine/src/vpu/graph_transformer/src/stages/loop_start.cpp

```diff
@@ -70,8 +71,53 @@ class LoopStart : public StageNode {
 
 }  // namespace
 
+namespace {
+
+class LoopDynamicIterNumber : public StageNode {
+public:
+    using StageNode::StageNode;
+
+protected:
+    StagePtr cloneImpl() const override {
+        return std::make_shared<LoopDynamicIterNumber>(*this);
+    }
+
+    // TODO: should this method be empty?
```

Comment:
I think we need to take a look at other stages. This one does not affect input in any way and even does not have an output - so it should look like other stage's implementation where stage has outputs of the same form as inputs.

The thing that we should care about is how output data is consumed - the only reasonable consumer is `LoopStart` and it does not take this output data into consideration as far as I can see - so I think it may be okay to leave it empty, but probably the reason should be mentioned in comment